src/components/PostList/PostList.jsx

```diff
@@ -1,3 +1,11 @@
-export const PostList = () => (
-  <>Put the list here</>
+import { PostInfo } from "../PostInfo";
+import './PostList.scss';
+
+
+export const PostList = ({ posts }) => (
+  <div className="PostList">
+    {posts.map(post => (
+      <PostInfo post={post} key={post.id}/>
```

Comment:
In your map function, you are directly passing the post object to the PostInfo component. This could lead to unnecessary rerenders if the post object has many properties. It would be more efficient to destructure the needed properties from the post object and pass those as props to the PostInfo component.